websites/B/Bungie/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "../../../tsconfig.json",
+  "compilerOptions": {
+    "lib":["DOM", "ES2017", "ES7"],
+    "outDir": "./dist"
+  }
+}
```

Comment:
https://docs.premid.app/en/dev/presence#filling-in-the-tsconfigjson-file

Please don't edit the tsconfig, you have to use the one we provide.

Reply:
I fixed that.